doc/whats_new/v1.5.rst

```diff
@@ -47,6 +47,9 @@ See :ref:`array_api` for more details.
 
 **Classes:**
 
+- :class:`sklearn.preprocessing.LabelBinarizer` now supports Array API compliant inputs.
+  :pr:`28626` by :user:`Jérôme Dockès <jeromedockes>`.
```

Comment:
Since we cut 1.5.X earlier this week, this entry will have to be pushed to `v1.6.rst` (after a merge of `main` into this branch).